utils/_context/_scenarios.py

```diff
@@ -958,6 +961,12 @@ def close_targets(self):
         logger.info(f"Pulumi stack down")
         self.stack.destroy(on_output=logger.info)
 
+    def customize_feature_parity_dashboard(self, result):
+        for test in result["tests"]:
+            last_index = test["path"].rfind("::") + 2
+            test["path"] = test["path"][last_index:]
+            logger.info(f"CUSTOMIZE Test: {test}")
```

Comment:
could you just add a comment explaining the why ?

Reply:
Because I can’t see the most relevant information: the name of the test + machine
<img width="846" alt="Screenshot 2024-02-06 at 14 26 05" src="https://github.com/DataDog/system-tests/assets/108007532/d5004ac2-cad7-47d0-8136-7cea32ad6be0">
Upsss...I'll remove the log too